contrib/ohi-release-notes/run.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+set -euo pipefail
+
+RT_PKG="github.com/newrelic/release-toolkit@latest"
+ARGS="$*"
+
+
+# creating a temporary folder where to build the rt binary and cleaning after exiting.
+TEMP_BIN=$(mktemp -dt release-toolkit-XXX)
+function cleanup() {
+    rm -rf $TEMP_BIN    || true
+    rm "${GIT_ROOT}/CHANGELOG.md.bak" || true
+    rm "${GIT_ROOT}/changelog.yaml"   || true
+}
+trap cleanup EXIT
+
+
+# usage and helm command. It is also an error exit in case flags are not correct.
+function help() {
+    set +x  # Disable verbosity. If it is enabled at this point, it is not needed anymore.
+    ERRNO=0
+    if ! [ -z "${1:-}" ]; then
+        echo "ERROR:"
+        echo "   $1"
+        echo ""
+        ERRNO=1
+    fi
+
+    cat <<EOM
+NAME:
+   $0 - release toolkit wrapper to create release notes for OHIs
+
+USAGE:
+   $0 [options]
+
+DESCRIPTION:
+   Wrapper for release toolkit that runs commands needed to create the release notes for an OHI:
+    * rt validate-markdown
+    * rt generate-yaml
+    * rt is-held
+    * rt link-dependencies
+    * rt update-markdown (with the version calculated from the next-version command)
+    * rt render-changelog (with the version calculated from the next-version command)
+
+   At the end of the run, this command should output two files and a string:
+    * CHANGELOG.md updated with the last changelog rendered.
+    * A CHANGELOG.partial.md with the changes for this release only.
+    * The version that was computed for this release.
+
+OPTIONS:
+   --git-root       Run all the command using this path as root
+   --verbose        Adds verbose mode to this script.
+   --help           Show this help message and exits.
+   --excluded-dirs  Exclude commits whose changes only impact files in specified dirs relative to repository root. Defaults to ".github".
+   --no-fail        Do not fail even in the held toggle is active
+   --dictionary     Sets the link dependency dictionary. Defaults to ".github/rt-dictionary.yaml".
+
+EOM
+    exit $ERRNO
+}
+
+
+# parsing flags
+EXCLUDED_DIRECTORIES=".github"
+IS_HELD_FAIL="--fail"
+DICTIONARY=".github/rt-dictionary.yaml"
+GIT_ROOT="."
+
+while true; do
+    if [ -z "${1:-}" ]; then
+        break;
+    else
+        case "${1}" in
+            -v | --verbose ) set -x; echo "Called with these arguments: $ARGS"; shift ;;
+            -h | --help ) help ;;
+            # Flags for all
+            --git-root ) GIT_ROOT="$2"; shift 2 ;;
+            # Flags for generate-yaml
+            --excluded-dirs ) EXCLUDED_DIRECTORIES="$2"; shift 2 ;;
+            # Flags for is-held
+            --no-fail ) IS_HELD_FAIL=""; shift ;;
+            # Flags for link-dependencies
+            --dictionary ) DICTIONARY="$2"; shift 2 ;;
+            * ) help "option is not accepted/parsable: \"$1\"" ;;
+        esac
+    fi
+done
+
+# building rt
+GOBIN="${TEMP_BIN}" go install ${RT_PKG}
```

Comment:
if I am not mistaken when you run an action you download the whole folder, should it try to run the same code? Otherwise we could end up running @latest instead of @v1

Reply:
This is [completely related to the previous message you left](https://github.com/newrelic/release-toolkit/pull/135#discussion_r1104163193) so I am closing this conversation.